WordPress/WordPressApi/WordPressComRestApi.swift

```diff
@@ -272,6 +247,163 @@ public class WordPressComRestApi: NSObject
         }
         return WordPressComRestApi.pathByAppendingPreferredLanguageLocale(path)
     }
+
+    private func URLForTemporaryFile() -> NSURL {
+        let fileName = "\(NSProcessInfo.processInfo().globallyUniqueString)_file.xmlrpc"
+        let fileURL = NSURL.fileURLWithPath(NSTemporaryDirectory()).URLByAppendingPathComponent(fileName)
+        return fileURL
+    }
+
+    private func multipartRequestWithURLString(urlString:String, parameters: [String:AnyObject], fileParts: [FilePart], encodedToFileURL fileURL:NSURL) -> NSURLRequest? {
+        //let urlStringWithLocale = appendLocaleIfNeeded(urlString)
+        guard let baseURL = NSURL(string: WordPressComRestApi.apiBaseURLString),
+              let url = NSURL(string:urlString, relativeToURL:baseURL)
+        else {
```

Comment:
Can we move the else to the upper line? 

(Noted elsewhere as well: in this case i just promote the `{` to a new line, no ideas if we have a style guide for that?)